src/runtime/analytics-service.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright 2018 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {feArgs, feUrl} from './services';
+import {createElement} from '../utils/dom';
+
+/** @const {!Object<string, string>} */
+const iframeStyles = {
+  display: 'none',
+};
+
+export class AnalyticsService {
+  /**
+   * @param {!../model/doc.Doc} doc
+   * @param {!web-activities/activity-ports.ActivityPorts} activityPorts
+   * @param {!../model/page-config.PageConfig} config
+   */
+  constructor(doc, activityPorts, config) {
+
+    /** @private @const {!Doc} */
+    this.doc_ = doc;
+
+    /** @private @const {!web-activities/activity-ports.ActivityPorts} */
+    this.activityPorts_ = activityPorts;
+
+    /** @private @const {!HTMLIFrameElement} */
+    this.iframe_ =
+        /** @type {!HTMLIFrameElement} */ (createElement(
+            this.doc_.getWin().document, 'iframe', iframeStyles));
```

Comment:
I double-checked the `createElement` function and the 3rd argument there is element attributes, _not_ styles. So, I think this is incorrect. Instead, probably we need this:

```
this.iframe_ = createElement(...document, 'iframe');
setImportantStyles(iframeStyles);
```